libs/providers/flagd/src/lib/service/common/grpc-util.ts

```diff
@@ -14,3 +16,18 @@ export const closeStreamIfDefined = (stream: ClientReadableStream<unknown> | und
     stream.destroy();
   }
 };
+
+/**
+ * Creates gRPC channel credentials based on TLS and certificate path configuration.
+ * @returns Channel credentials for gRPC connection
+ */
+export const createChannelCredentials = (tls: boolean, certPath?: string) => {
```

Comment:
Bit of nit picking: Is it possible to provide an explicit return type for this function?